sci-libs/opencascade/opencascade-7.5.1-r1.ebuild

```diff
@@ -80,6 +80,13 @@ src_prepare() {
 
 	sed -e 's/\/lib\$/\/'$(get_libdir)'\$/' \
 		-i adm/templates/OpenCASCADEConfig.cmake.in || die
+
+	# There is an OCCT_UPDATE_TARGET_FILE cmake macro that fails due to some
+	# assumptions it makes about installation paths. Rather than fixing it, just
+	# get rid of the mechanism altogether - its purpose is to allow a
+	# side-by-side installation of release and debug libraries.
+	sed -e 's|\\${OCCT_INSTALL_BIN_LETTER}||' \
+		-i "adm/cmake/occt_toolkit.cmake"
```

Comment:
Don't we need to cover `adm/cmake/occt_macros.cmake` as well with this statement? A `find` leads me to these two files with `OCCT_INSTALL_BIN_LETTER` in it:
```
waebbl $ find . -name *.cmake -exec grep -H OCCT_INSTALL_BIN_LETTER {} \;
./adm/cmake/occt_toolkit.cmake:           DESTINATION "${INSTALL_DIR_BIN}\${OCCT_INSTALL_BIN_LETTER}")
./adm/cmake/occt_toolkit.cmake:    install (FILES  ${CMAKE_BINARY_DIR}/${OS_WITH_BIT}/${COMPILER}/bin\${OCCT_INSTALL_BIN_LETTER}/${PROJECT_NAME}.pdb
./adm/cmake/occt_toolkit.cmake:             DESTINATION "${INSTALL_DIR_BIN}\${OCCT_INSTALL_BIN_LETTER}")
./adm/cmake/occt_toolkit.cmake:           RUNTIME DESTINATION "${INSTALL_DIR_BIN}\${OCCT_INSTALL_BIN_LETTER}"
./adm/cmake/occt_toolkit.cmake:           ARCHIVE DESTINATION "${INSTALL_DIR_LIB}\${OCCT_INSTALL_BIN_LETTER}"
./adm/cmake/occt_toolkit.cmake:           LIBRARY DESTINATION "${INSTALL_DIR_LIB}\${OCCT_INSTALL_BIN_LETTER}")
./adm/cmake/occt_toolkit.cmake:      set (LINK_NAME    "${INSTALL_DIR}/${INSTALL_DIR_LIB}\${OCCT_INSTALL_BIN_LETTER}/lib${PROJECT_NAME}${CMAKE_SHARED_LIBRARY_SUFFIX_DEFAULT}")
./adm/cmake/occt_toolkit.cmake:      set (LIBRARY_NAME "${INSTALL_DIR}/${INSTALL_DIR_LIB}\${OCCT_INSTALL_BIN_LETTER}/lib${PROJECT_NAME}${CMAKE_SHARED_LIBRARY_SUFFIX}")
./adm/cmake/occt_macros.cmake:# Adds OCCT_INSTALL_BIN_LETTER variable ("" for Release, "d" for Debug and 
./adm/cmake/occt_macros.cmake:      string (REGEX REPLACE \"[\\\\]?[\\\$]{OCCT_INSTALL_BIN_LETTER}\" \"\${OCCT_INSTALL_BIN_LETTER}\" line \"\${line}\")
./adm/cmake/occt_macros.cmake:    set (OCCT_INSTALL_BIN_LETTER \"\")
./adm/cmake/occt_macros.cmake:    set (OCCT_INSTALL_BIN_LETTER \"i\")
./adm/cmake/occt_macros.cmake:    set (OCCT_INSTALL_BIN_LETTER \"d\")
```

Reply:
Did a check of the code. Looks like it's ok to only strip it from `occt_toolkit.cmake`. The `occt_macros.cmake` only contains the macro definition and another macro `OCCT_INSERT_CODE_FOR_TARGET` which uses the `OCCT_INSTALL_BIN_LETTER` macro, which is only used with `if(MSVC)` guards and therefore not relevant for Gentoo users.

Reply:
Thanks, it'd be easy to just add a 2nd line in `sed -i` :)